test/watermelon-spec.js

```diff
@@ -0,0 +1,34 @@
+'use strick';
+
+const algolib = require('../index.js');
+
+describe('#watermelon', function () {
+
+    it('make decision if weight<4', function () {
+        let decision = algolib.watermelon.makeDecision(3);
+        decision.should.be.eql("NO");
+    });
+
+
+    it('make decision if weight=1', function () {
+        let decision = algolib.watermelon.makeDecision(1);
+        decision.should.be.eql("NO");
+    });
+
+    it('make decision if weight=150', function () {
+        let decision = algolib.watermelon.makeDecision(150);
+        decision.should.be.eql("YES");
+    });
+
+    it('make decision if weight is even', function () {
```

Comment:
I checked range limits of input values, odd and even numbers. Previous is a limit of input range.